packages/grid_client/scripts/connection.ts

```diff
@@ -0,0 +1,64 @@
+/* 
+  Use this script to test grid connection:
+  This script creates 2 instances of the grid client, then after 10 minutes, it performs a query on both instances.
+
+  Steps:
+    1 - Connect to the grid
+    2 - Close the internet connection
+    3 - After 9 minutes, open the internet connection again
+  The grid should automatically reconnect and request the twin ID.
+*/
+
+import { GridClient, NetworkEnv } from "../src";
+
+// Replace with actual mnemonics
+const clientMnemonic1 = "your_mnemonic_here";
+const clientMnemonic2 = "your_mnemonic_here";
+
+// Choose the desired network (main, test, qa, dev)
+const NETWORK = NetworkEnv.dev;
+
+// Function to log the current minute along with the client's name
+function logCurrentMinute(clientName: string, time = 1) {
+  console.log(`${clientName} running count ${time}`);
```

Comment:
what are you counting on here? the log will log like 
`Client: 1 running count 1`

Reply:
No will log 1 if I don't pass the time, otherways will log the actual time, I want to set this as optional because I don't wanna send it at the beginning of the script.

Reply:
I mean here that the message doesn't tell me what are we counting

Reply:
updated to be like that `Client 1 is in its 3 execution cycle.`